websites/views.py

```diff
@@ -36,25 +40,49 @@ class WebsiteViewSet(
 
     serializer_class = WebsiteSerializer
     pagination_class = DefaultPagination
-    permission_classes = (ReadonlyPermission,)
+    permission_classes = (HasWebsitePermission,)
 
     def get_queryset(self):
-        """ Generate a QuerySet for fetching published websites """
-        ordering = self.request.query_params.get("sort", "-publish_date")
+        """
+        Generate a QuerySet for fetching websites.
+        Anonymous users should get a list of all published websites (used for ocw-www carousel)
+        Global admins should get a list of all websites, published or not.
+        Other authenticated users should get a list of websites they are editors/admins/owners for.
```

Comment:
Nit: I think these 3 lines should be inline comments, located above the `if` clauses or above where the `queryset` values are set. If the querysets are changed in the body of this method, we are much more likely to forget to update this docstring